### ISSUE TITLE
Radcliffe 2: Contact info can't be deleted from header or footer

### ISSUE THREAD

**Issue by dromero20** [Automattic/themes#492]
## Steps to replicate

Only happening on this site https://davidtaborn.com, so I couldn't replicate it in my test one.

- Go to Customizer -> Contact Info
- Delete contact info information and click on Publish (it's deleted on the preview).
- Go to the site (info still in there).
- Reload Customizer, the contact info popped up again.

## Expected

According with my tests, when you remove contact info from Customizer, those fields should be completely deleted.

* User report: #6743578-hc
* Follow-up ticket: #1720124-zd
* User's site: https://davidtaborn.com

**Comment by laurelfulford** [comment]
This is a weird one, and similar to this issue: p2EDhh-Cj-p2.

I've manually cleared some theme_mods for this site and it looks like it fixed the issue, so I'm going to close it. I'm not sure what the underlying cause is, but we have something similar happen from time to time with color mods.

**Comment by laurelfulford** [comment]
I did a little digging into this issue -- I this this was introduced in r52410-wpcom-themes; those files got a little out of sync with the GitHub repo, so I've committed those changes in #587 to get things back in sync.

It looks like the `theme_mods` were switched to options, with a fallback to the `theme_mod`. I think this causes an issue on sites that were created prior to that switch, if the contact info is already stored as a `theme_mod`.

Changing any of the values to something new will be saved as an `option` rather than overwriting the original `theme_mod`. Emptying one of the values will empty the `option`, but that will cause the function that outputs the contact information to display the still-there `theme_mod` as a fallback, here:

https://github.com/Automattic/themes/blob/37d47c445bf914f91ba5b00dc5f9bc8e09c24e0d/radcliffe-2/contact-info/contact-info-functions.php#L8-L10

I'm going to keep looking into this!

**Comment by crunnells** [comment]
@laurelfulford Could this be related to #502?

**Comment by laurelfulford** [comment]
@crunnells I don't think so -- only the contact inputs were changes to `option`s, but the style packs are still stored as `theme_mod`s.

We have had some weirdness around style packs that stems from the order different customizations are applied -- and the confusion over what is coming from a style pack, and what is coming from the custom colours. There are a couple other issues logged here: https://github.com/Automattic/style-packs-theme/

It's hard to tell exactly what's happening in #502 without a bit more information 😕

**Comment by ramonjd** [comment]
@laurelfulford We have a task on our Trello board to help look at this. I've tried to replicate this on a sandboxed site and on davidtaborn.com, but couldn't. 🤷‍♂️ 

https://cldup.com/-ascV-FMnj.mp4

Aaaand I've run out of ideas. :) 

I'm probably missing something, but not terribly well-versed. Let us know if we can help further.

**Comment by laurelfulford** [comment]
Thanks for looking at this @ramonjd! 

It's a bit of a weird issue -- you won't be able to recreate it on a new site now, and I've manually cleared the `theme_mod`s on davidtaborn.com to fix it there. 

It's happening on sites that were created before the contact information was switched from being stored as a `theme_mod` to being stored as an `option`.

So steps to recreate might look like:

1. Start with a site running Radcliffe 2 - might be good to start with a fresh one.
2. Revert the changes from r52410-wpcom-themes (if it's a WP.com sandbox) or #587 (if it's on localhost). This will switch the contact info fields in the theme back to using `theme_mod`s.
3. Add something to the contact fields, and Publish changes.
4. Undo the revert so you're now running the current version of the theme, where the contact info has been moved to options.
5. Try to remove the contents of one or all of the contact info fields, and publish changes.

The update should preview as expected in the Customizer, but when you view the live site the contact information you removed will still be there. And when you go back into the Customizer, the fields will be populated again.

Just let me know if you have any questions at all about this!
